autoprotocol/util.py

```diff
@@ -213,3 +213,22 @@ def _check_container_type_with_shape(container_type, shape):
             f"a full row consists of {format_columns} columns and a full "
             f"column consists of {format_rows} rows, but {shape} was specified."
         )
+
+
+def _validate_liha_shape(device: str, shape: dict) -> None:
+    """Validates LiHa shape for liquid handle dispense"""
```

Comment:
i got chatgpt to make a docstring for us. can you edit and include this?

```suggestion
    """Validates LiHa shape for liquid handle dispense.

    Args:
        device (str): The LiHa device name. Currently supported devices are 
                      "x_mantis" and "x_tempest_chip".
        shape (dict): A dictionary containing the number of rows and columns
                      for the LiHa shape. For example, {"rows": 8, "columns": 1}.

    Raises:
        ValueError: If the provided device name is not one of the supported 
                    devices or if the provided shape is not valid for the 
                    specified device.

    Returns:
        None

    Example:
        >>> _validate_liha_shape("x_mantis", {"rows": 1, "columns": 1})
        None
    """
```

Reply:
LOL love, committing this suggestion